frontend/src/components/mspcontracts/BeneficiaryAddress.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <tr>
+    <td>Home Address</td>
+    <td>{{ result.homeAddressLine1 }}</td>
+    <td>{{ setHomeAddressLine2 }}</td>
+    <td>{{ parseHomeAddress }}</td>
+    <td>{{ result.homeAddressPostalCode }}</td>
+    <td>{{ result.telephone }}</td>
+  </tr>
+  <tr>
+    <td>Mailing Address</td>
+    <td>{{ result.mailingAddressLine1 }}</td>
+    <td>{{ setMailingAddressLine2 }}</td>
+    <td>{{ parseMailingAddress }}</td>
+    <td>{{ result.mailingAddressPostalCode }}</td>
+    <td>{{ result.telephone }}</td>
+  </tr>
+</template>
+<script>
+export default {
+  props: {
+    result: Object,
+  },
+  computed: {
+    setHomeAddressLine2() {
+      return !this.result.homeAddressLine3 && this.result.homeAddressLine2 ? '' : this.result.homeAddressLine2
```

Comment:
I feel like this might read be better if a) there were brackets around the OR condition and b) the condition was reversed. I.e. IF (condition) THEN show address line 2 ELSE blank.